script.js

```diff
@@ -1,21 +1,299 @@
+        ['Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', 'Del'],
+        ['Capslock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '\'', 'Enter'],
+        ['Shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', '↑', 'Shift'],
+        ['Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl']]
+    },
+
+    spec_symbol: {
+        rus: {
+            'Digit1': '!', 'Digit2': '\"', 'Digit3': '№', 'Digit4': ';', 'Digit5': '%',
+            'Digit6': ':', 'Digit7': '?', 'Digit8': '*', 'Digit9': '(', 'Digit0': ')', 'Minus': '_', 'Equal': '+', 'Slash': ','
+        },
+
+        eng: {
+            'Backquote': '~', 'Digit1': '!', 'Digit2': '@', 'Digit3': '#', 'Digit4': '$', 'Digit5': '%',
+            'Digit6': '^', 'Digit7': '&', 'Digit8': '*', 'Digit9': '(', 'Digit0': ')', 'Minus': '_', 'Equal': '+', 'Slash': '?', 'BracketLeft': '{',
+            'BracketRight': '}', 'Semicolon': ':', 'Quote': '"', 'Comma': '<', 'Period': '>'
+        },
+
+    }
+
+
+}
+
+let caps = false;
+
+const keyCode = [['Backquote', 'Digit1', 'Digit2', 'Digit3', 'Digit4', 'Digit5', 'Digit6', 'Digit7', 'Digit8', 'Digit9', 'Digit0', 'Minus', 'Equal', 'Backspace'],
+['Tab', 'KeyQ', 'KeyW', 'KeyE', 'KeyR', 'KeyT', 'KeyY', 'KeyU', 'KeyI', 'KeyO', 'KeyP', 'BracketLeft', 'BracketRight', 'Delete'],
+['CapsLock', 'KeyA', 'KeyS', 'KeyD', 'KeyF', 'KeyG', 'KeyH', 'KeyJ', 'KeyK', 'KeyL', 'Semicolon', 'Quote', 'Enter'],
+['ShiftLeft', 'KeyZ', 'KeyX', 'KeyC', 'KeyV', 'KeyB', 'KeyN', 'KeyM', 'Comma', 'Period', 'Slash', 'ArrowUp', 'ShiftRight'],
+['ControlLeft', 'MetaLeft', 'AltLeft', 'Space', 'AltRight', 'ArrowLeft', 'ArrowDown', 'ArrowRight', 'ControlRight']];
+
 
 let container = document.createElement('div');
 container.className = 'container';
 document.body.append(container);
-let textArea=document.createElement('textarea');
+let textArea = document.createElement('textarea');
 textArea.className = 'textarea-style';
-container.appendChild   (textArea);    
+container.appendChild(textArea);
+
+//create buttons
+function createkeyboard(inerText) {
+    for (let i = 0; i < inerText.length; i++) {
+        let line = document.createElement('divv');
+        line.classList.add("button-line-style");
+        container.appendChild(line);
+        for (let j = 0; j < inerText[i].length; j++) {
+            let btn = document.createElement('div');
+            btn.classList.add(style(keyCode[i][j]));
+            btn.id = keyCode[i][j];
+            btn.innerText = inerText[i][j];
+            line.appendChild(btn);
+        }
+
+    }
+}
+
+// change div inerText
+function refresh(str) {
+    if (str == "rus") {
+        container.querySelectorAll('div').forEach(function (e, i) {
+            e.innerText = Keyboard.inerText.rus.flat()[i];
+        });
+    }
+    if (str == "eng") {
+        container.querySelectorAll('div').forEach(function (e, i) {
+            e.innerText = Keyboard.inerText.eng.flat()[i];
```

Comment:
line 71 and 76 looks same with little diffrence (language), better use function for it